src/couch/src/couch_util.erl

```diff
@@ -41,6 +41,7 @@
 -export([set_mqd_off_heap/1]).
 -export([set_process_priority/2]).
 -export([hmac/3]).
+-export([unquote/1]).
```

Comment:
I think it should be called `unquote_path`. Because it deals with a corner case and suitable for generic application.

Reply:
I agree. It is easier to follow with a universal name. Fixed in https://github.com/apache/couchdb/pull/3638/commits/48caeb15b15a006f85ed4bc2f2606cb3989ba44a.